ssl/build.info

```diff
@@ -37,8 +37,11 @@ IF[{- !$disabled{'deprecated-3.0'} -}]
   SHARED_SOURCE[../libssl]=s3_cbc.c
   SOURCE[../libssl]=ssl_rsa_legacy.c
 ENDIF
-
```

Comment:
This one looks a little funny

Reply:
Not incorrect, but strange how it was removed. I can put it back in.